Sources/Sentry/SentryTracer.m

```diff
@@ -109,7 +109,12 @@ - (void)captureTransaction
 {
     NSArray *spans;
     @synchronized(_spans) {
-        spans = [_spans copy];
+        spans = [_spans
+            filteredArrayUsingPredicate:[NSPredicate
```

Comment:
`l`: It would be nice to add a comment on why we do this.

Reply:
Sure, good idea.